types/components/ToggleButtonGroup.d.ts

```diff
@@ -14,8 +14,8 @@ export interface ToggleButtonRadioProps<T> {
 export interface ToggleButtonCheckboxProps<T> {
   type: 'checkbox';
   name?: string;
-  value?: T;
-  defaultValue?: T;
+  value?: T[];
+  defaultValue?: T[];
```

Comment:
wait no, these can be `T` or `T[]`, depending on whether the type is `checkbox` or `radio`

Reply:
Checkbox and Radio Props have separate typings. The generalized typing is a union of those two. Is the radio supposed to be single-value and the checkbox multi-value? In that case, only the checkbox would have T[]. That is from my understanding at least.

Reply:
yup those all come from `ToggleButtonRadioProps`

Reply:
ha sorry, didn't notice that in the github diff view. tbf tho, the component always allows T or T[] regardless of type, but i'd agree it's better to have the types be stricter